hieradata_aws/class/integration/ckan_db_admin.yaml

```diff
@@ -11,39 +11,15 @@ govuk_env_sync::tasks:
     temppath: "/tmp/ckan_production"
     url: "govuk-production-database-backups"
     path: "postgresql-backend"
-  # "push_ckan_production_daily":
-  #   ensure: "present"
-  #   hour: "5"
-  #   minute: "0"
-  #   action: "push"
-  #   dbms: "postgresql"
-  #   storagebackend: "s3"
-  #   database: "ckan_production"
-  #   database_hostname: "ckan-postgres"
-  #   temppath: "/tmp/ckan_production"
-  #   url: "govuk-staging-database-backups"
-  #   path: "ckan-postgres"
-  "pull_ckan_pycsw_production_daily":
+  "push_ckan_production_daily":
     ensure: "present"
-    hour: "0"
+    hour: "5"
     minute: "0"
-    action: "pull"
+    action: "push"
     dbms: "postgresql"
     storagebackend: "s3"
-    database: "ckan_pycsw_production"
+    database: "ckan_production"
     database_hostname: "ckan-postgres"
-    temppath: "/tmp/ckan_pycsw_production"
-    url: "govuk-production-database-backups"
-    path: "postgresql-backend"
-  # "push_ckan_pycsw_production_daily":
-  #   ensure: "present"
-  #   hour: "5"
-  #   minute: "0"
-  #   action: "push"
-  #   dbms: "postgresql"
-  #   storagebackend: "s3"
-  #   database: "ckan_pycsw_production"
-  #   database_hostname: "ckan-postgres"
-  #   temppath: "/tmp/ckan_pycsw_production"
-  #   url: "govuk-staging-database-backups"
-  #   path: "ckan-postgres"
+    temppath: "/tmp/ckan_production"
+    url: "govuk-staging-database-backups"
```

Comment:
I'm confused by this task being called "push_ckan_production_daily" but using a url of "govuk-staging-database-backups" and being in the integration hieradata...

Should this be production?

Reply:
unfortunately the name of the database is production, I'll update the name to make it a bit clearer